modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/IgnitePlanner.java

```diff
@@ -165,6 +191,7 @@ private CalciteConnectionConfig connConfig() {
         open = false;
     }
 
+    /** */
     private void ready() {
```

Comment:
Planner implements an interface what have `reset()` method, after this method call we need to clear anything except base config. The `ready()` method guaranties all needed parts are prepared before them are used even in case the planner was reset.